inst/json/metadata_0.1.0.json

```diff
@@ -75,9 +75,11 @@
       { "id": "business_closures", "label": "Business closures" }
     ],
     "time_series": [
-      { "id": "cases", "label": "Cases", "description": "Number of confirmed cases" },
-      { "id": "vacc", "label": "Vaccinated", "description": "Cumulative vaccinations" },
-      { "id": "hosp_occ", "label": "Hospital Occupancy", "description": "Number of hospitalised cases" }
+      { "id": "infect_asymp", "label": "Asymptomatic Infections", "description": "Infections without symptoms" },
+      { "id": "infect_symp", "label": "Symptomatic Infections", "description": "Infections with symptoms" },
+      { "id": "hospitalised", "label": "Hospital Occupancy", "description": "Number of hospitalised cases" },
+      { "id": "recovered", "label": "Recovered", "description": "Recovered from infection" },
+      { "id": "dead", "label": "Dead", "description": "Cumulative deaths" }
     ]
   }
 }
```

Comment:
I've noticed that in this metadata json, we are inconsistent about case - we mix `My Label` and `My label`. My preference is for the latter: capitalise only the first word.